internal/db/schema/migrations/oss/postgres/62/01_job_run_index.up.sql

```diff
@@ -0,0 +1,7 @@
+begin;
+
+  delete from job_run where status='completed';
+
+  create index on job_run (status);
```

Comment:
Most of the index's we create, apart from 1 that I found create a named index. https://github.com/hashicorp/boundary/blob/main/internal/db/schema/migrations/oss/postgres/50/01_session_list_index.up.sql Often we also add a comment on the index https://github.com/hashicorp/boundary/blob/main/internal/db/schema/migrations/oss/postgres/10/04_vault_credential.up.sql#L182

Reply:
Good shout, I added a comment and a name.